packages/core/src/Stores/notification-store.js

```diff
@@ -1213,6 +1223,53 @@ export default class NotificationStore extends BaseStore {
                     text: localize('Personal details'),
                 },
             },
+            poo_required: {
+                key: 'poo_required',
+                header: (
+                    <Localize
+                        i18n_default_text='<0>Proof of ownership</0> <1>required</1>'
+                        components={[<div key={0} />, <div key={1} />]}
+                    />
+                ),
+                message: (
+                    <Localize
+                        i18n_default_text='<0></0><1>Your account is currently locked</1> <2></2><3>Please upload your proof of</3> <4>ownership to unlock your account.</4> <5></5>'
+                        components={[
+                            <br key={0} />,
```

Comment:
why do we need key for these?

Reply:
Yes @ako, If you look closely, you'll see that the `components` prop takes in an `Array`